adafruit_rfm9x.py

```diff
@@ -273,9 +275,10 @@ def __init__(
         self.signal_bandwidth = 125000
         self.coding_rate = 5
         self.spreading_factor = 7
-        # Default to disable CRC checking on incoming packets.
-        self.enable_crc = False
-        # set AGC - Default = True
+        # Default to enable CRC checking on incoming packets.
+        self.enable_crc = crc
```

Comment:
Previously, the `enable_crc` defaults to `False`, it's currently set to `True`. Is this a mistake?

Reply:
That was the purpose of the PR -- to set ti tp True by default -- to match Arduino RadioHead

Reply:
I just added it as a kwarg since it seems like a good thing to do.

Reply:
ok! thank you for the clarification